dandi/cli/cmd_download.py

```diff
@@ -1,10 +1,10 @@
 import os
 
 import click
-from .command import devel_option, main, map_to_click_exceptions
+from .base import devel_option, map_to_click_exceptions
 
 
-@main.command()
+@click.command()
```

Comment:
shouldn't it be a part of main to take advantage of all of its setup etc?

Wouldn't we just move everything BUT following
```
#
# Commands in the main group
#
from .cmd_ls import ls  # noqa: E402, F401
from .cmd_organize import organize  # noqa: E402, F401
from .cmd_upload import upload  # noqa: E402, F401
from .cmd_download import download  # noqa: E402, F401
from .cmd_validate import validate  # noqa: E402, F401
from .cmd_register import register  # noqa: E402, F401
```
from `command.py` into `base.py` and adjust the imports to use `from .base`?  alternative could be just to move those lines into e.g. `cli/ui.py` and import that one upstairs where command is imported now?

Reply:
Take advantage of what setup?  Using `@main.command` vs. `@click.command` + `main.add_command()` should make no difference.  Deciding between how I organized the command imports versus your suggestions seems like just a stylistic choice.

Reply:
I have missed `main.add_command()` invocations... ok, let's proceed with your stylistic choice.